### ISSUE TITLE
Increase character limit of activity title to 120

### ISSUE THREAD

**Issue by ralphboeije** [part-up/part-up#1752]

**Comment by noelheesen** [comment]
I disagree, a title should be short. If you really want this then how many characters should be shown in the boardview as 120 won't fit?

**Comment by ralphboeije** [comment]
hard to argue the right size here. I also understand your point, so let's hold this for a moment. I'll double check with the client.
